### ISSUE TITLE
hyper-parameters for domainbed

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#212]
https://arxiv.org/pdf/2007.01434.pdf
appendix D2

Condition Parameter Default value Random distribution
ResNet
learning rate 0.00005 10Uniform(−5,−3.5)
batch size 32 2
Uniform(3,5.5)
generator learning rate 0.00005 10Uniform(−5,−3.5)
discriminator learning rate 0.00005 10Uniform(−5,−3.5)

**Comment by smilesun** [comment]
[5b03981](https://github.com/marrlab/DomainLab/commit/5b039812f427c707a431a586cc1140e0e5b137d9)
